spec.html

```diff
@@ -539,6 +539,11 @@
           [+N] SourceCharacter but not one of `c` or `k`
         </ins>
 
+        ClassAtomNoDash[U] ::
+          SourceCharacter but not one of `\` or `]` or `-`
+          `\` ClassEscape[?U, <ins>?N</ins>]
```

Comment:
If you're passing through ?N, presumably it also need to be listed as an argument, right?

Reply:
Right! Fixed.